packages/webapp/components/layouts/MainFeedPage.tsx

```diff
@@ -27,7 +30,10 @@ export type MainFeedPageProps = {
   isFinder?: boolean;
 } & Pick<MainFeedLayoutProps, 'searchChildren'>;
 
-const getFeedName = (path: string): string => {
+const getInternalFeedName = (
```

Comment:
Renaming because it confused me which one was called for 10 minutes (same as above import)